### DISCUSSION TITLE
How to Improve Fine-Tuning Accuracy

### DISCUSSION THREAD

**Discussion by seonhyep** [CederGroupHub/chgnet#191] [Q&A]
When I compared the energy values predicted by CHGNet after fine-tuning with the energy values from AIMD and plotted the graph, the overall shape of the graph was the same as when I used the pretrained model, with only the slope and position changing. Is there a fine-tuning method that can change the overall shape of the graph? 
[figure.zip](https://github.com/user-attachments/files/16689476/figure.zip)
Also, I tried several approaches to train materials that are not in the Materials Project database, but the MAE value was very high, and the graph comparing it with AIMD values did not even show a linear relationship. Are there any ways to improve the training process?
